process/peer/process.go

```diff
@@ -880,19 +897,29 @@ func (vs *validatorStatistics) decreaseAll(shardId uint32, missedRounds uint64,
 	return nil
 }
 
-func (vs *validatorStatistics) saveInitialValueForMap(nodesMap map[uint32][][]byte, startEpoch uint32, stakeValue *big.Int, startRating uint32) error {
-	for _, pks := range nodesMap {
-		for _, pk := range pks {
-			node, _, err := vs.nodesCoordinator.GetValidatorWithPublicKey(pk, startEpoch)
-			if err != nil {
-				return err
-			}
+// Process - processes a validatorInfo and updates fields
+func (vsp *validatorStatistics) Process(vid *state.ValidatorInfo) error {
```

Comment:
Can you rename the receiver to `vs` the warnings in this file dissapear?

Reply:
fixed